docs/source/upgrade/index.rst

```diff
@@ -82,13 +83,103 @@ has one *nonconsuming* choice that accepts a *Coin* contract, archives
 this *Coin* contract and creates a *CoinWithAmount* contract with
 *amount* set to 1.
 
-.. literalinclude:: daml/Upgrade.daml
+.. literalinclude:: example/coinV2/daml/UpgradeFromCoinV1.daml
   :language: daml
   :start-after: -- UPGRADE_AGREEMENT_BEGIN
   :end-before: -- UPGRADE_AGREEMENT_END
 
-.. TODO Turn this into a full example project with multiple packages
-.. and explain builds and deployment.
 
-.. Building and deploying
-.. ======================
+Building and deploying version-1
+================================
+
+Let's see everything in action by following the steps to build and deploy version-1 of our coin. After this we'll see how to deploy and upgrade to version-2.
+
+First we'll need a sandbox ledger to which we can deploy.
+
+.. code-block:: none
+
+   $ daml sandbox --port 7001
```

Comment:
Why not use the default port here?

Reply:
I prefer the explicitness. I think magic default port numbers are a pretty bad idea.

Reply:
That’s reasonable but can we explicitly specify the default then, i.e., `--port 6865`?